Sources/BezierSwift/MasterComponent/BezierToast/BezierToast.swift

```diff
@@ -0,0 +1,151 @@
+//
+//  BezierToast.swift
+//  
+//
+//  Created by dumba on 2023/02/10.
+//
+
+import SwiftUI
+
+struct BezierToastInfo: Equatable {
+  var leftImage: Image?
+  var leftImageTintColor: SemanticColor = .bgtxtAbsoluteWhiteDark
+  var title: String
+  
+  var leftImageLength: CGFloat { CGFloat(20) }
+  var leftImageTopPadding: CGFloat { CGFloat(3) }
+  
+  init(title: String) {
+    self.title = title
+  }
+  
+  init(leftImage: Image, title: String) {
+    self.leftImage = leftImage
+    self.title = title
+  }
+  
+  init(leftImage: Image, leftImageTintColor: SemanticColor, title: String) {
+    self.leftImage = leftImage
+    self.leftImageTintColor = leftImageTintColor
+    self.title = title
+  }
+  
+  static func == (lhs: Self, rhs: Self) -> Bool {
+    return false
+  }
+}
+
+struct BezierToast: View, Themeable {
+  private enum Metric {
+    static let contentHorizontalPadding = CGFloat(10)
+    
+    static let conentHStackSpacing = CGFloat(6)
+    static let contentStackVerticalPadding = CGFloat(10)
+    static let contentStackHorizontalPadding = CGFloat(14)
+
+    static let textTopPadding = CGFloat(4)
+  }
+  
+  private enum Constant {
+    static let contentMaxWidth = CGFloat(460)
+    static let textLineLimit = 2
+    static let animationDuration = CGFloat(0.3)
+    static let disappearDelay = CGFloat(3.0)
+  }
+  
+  @Environment(\.colorScheme) public var colorScheme
+  
+  @State private var isPresented: Bool = false
+  
+  private var info: BezierToastInfo
+  
+  init(info: BezierToastInfo) {
+    self.info = info
+  }
+  
+  var body: some View {
+    VStack(alignment: .center, spacing: .zero) {
+      if self.isPresented {
+        VStack {
+          HStack(alignment: .top, spacing: Metric.conentHStackSpacing) {
+            if let leftImage = self.info.leftImage {
+              leftImage
+                .resizable()
+                .foregroundColor(self.palette(self.info.leftImageTintColor))
+                .scaledToFit()
+                .frame(width: self.info.leftImageLength, height: self.info.leftImageLength)
+                .padding(.top, self.info.leftImageTopPadding)
+            }
+            
+            VStack {
+              Text(self.info.title)
+                .applyBezierFontStyle(.bold14, semanticColor: .bgtxtAbsoluteWhiteDark)
+                .lineLimit(Constant.textLineLimit)
+                .fixedSize(horizontal: false, vertical: true)
+                .padding(.vertical, Metric.textTopPadding)
+            }
+          }
+          .padding(.vertical, Metric.contentStackVerticalPadding)
+          .padding(.horizontal, Metric.contentStackHorizontalPadding)
+        }
+        .background(self.palette(SemanticColor.bgBlackDarker))
```

Comment:
```suggestion
        .background(self.palette(.bgBlackDarker))
```
형태로 앞쪽 전체적으로 생략 부탁드려용:)